package.json

```diff
@@ -19,7 +19,7 @@
     "discord.js": "^12.5.3",
     "dotenv": "^8.2.0",
     "i18n": "^0.13.2",
-    "keyv": "^3.1.0",
+    "keyv": "^4.0.3",
```

Comment:
You could have just let it at `"^4.0.0"` or `"4.x.x"` the minor version does not matter